chcemvediet/apps/anonymization/cron.py

```diff
@@ -0,0 +1,32 @@
+from django.core.files.base import ContentFile
+
+from poleno.cron import cron_job
+from poleno.attachments.models import Attachment
+from chcemvediet.apps.inforequests.models import Action
+
+from .models import AttachmentNormalization
+from . import content_types
+
+
+@cron_job(run_every_mins=1)
+def attachment_normalization():
+    attachment = Attachment.objects.attached_to(Action).filter(
+        attachmentnormalization__isnull=True
+    ).first()
+
+    if attachment is None:
+        return
+    if attachment.content_type == content_types.PDF_CONTENT_TYPE:
+        AttachmentNormalization.objects.create(
+            attachment=attachment,
+            successful=True,
+            file=ContentFile(attachment.content),
+            content_type=content_types.PDF_CONTENT_TYPE
+        )
```

Comment:
Este mi napadlo, ze by bolo fajn zapisat do logov, ze sa konverzia (ne)podarila. Po uspesnej nomalizacii pouzi `cron_logger` z `poleno.cron` za zalogovanie INFO logu `"Normalized attachment: {attachment}"`. Vid napriklad `poleno/mail/cron.py` ako sa `cron_logger` pouziva.

V neuspesnej vetve zaloguj INFO log `"Skipping normalization of attachment with not supported content type: {attachment}"`. Alebo nieco take.

V dalsich issues attachmenty normalizovane libreofficom zaloguj ako `"Normalized attachment using libreoffice: {attachment}"`. Podobne potom ak nomrlizacia failne, zaloguj ERROR log `"Normalizing attachment using libreoffice failed: {attachment}\n{trace}"`. Analogicky pri dalsich metodach normalizacie potom neskor.

Reply:
:ok: